lib/ansible/modules/crypto/openssl_pkcs12.py

```diff
@@ -222,7 +222,51 @@ def _check_pkey_passphrase():
         if not state_and_perms:
             return state_and_perms
 
-        return _check_pkey_passphrase
+        if os.path.exists(self.path) and module.params['action'] == 'export':
+            dummy = self.generate(module)
+            self.src = self.path
+            pkcs12_privatekey, pkcs12_certificate, pkcs12_other_certificates, pkcs12_friendly_name = self.parse()
```

Comment:
You should wrap this in a `try/except` and return `False` if it raises an exception.